content_scripts/mode_find.coffee

```diff
@@ -78,6 +78,7 @@ class FindMode extends Mode
     HUD.showFindMode this
 
   exit: (event) ->
+    HUD.giveUpFocus()
```

Comment:
Shouldn't this be `clearFocus`? Can you add a comment as to why focus needs to be cleared (reference #3409 if needed).

Reply:
Sorry `clearFocus` was just a typo. I've rebased the code and add comments.